src/app/components/shared/datetime/datetime/datetime.component.ts

```diff
@@ -0,0 +1,132 @@
+import { Component, Input } from "@angular/core";
+import { isInstantiated } from "@helpers/isInstantiated/isInstantiated";
+import { TimezoneInformation } from "@interfaces/apiInterfaces";
+import { DateTime, IANAZone, Zone } from "luxon";
+import { AbstractDatetimeComponent } from "../abstract-datetime.component";
+
+type BawTimezoneUnion = Zone | TimezoneInformation | string;
+
+/**
+ * Displays a datetime (possibly from another timezone) in the users locale and timezone
+ * Commonly used for displaying client side timestamps e.g. Project created at: xyz
+ *
+ * This component does not assume that all dates input are utc, and will localize using ISO8601 offset information if possible
```

Comment:
This comment seems redunant.

```suggestion
 * This component always shows a date in the user's  local timezone.
```